test/src/helpers/pointsTest.cpp

```diff
@@ -0,0 +1,168 @@
+#include <helpers/testPointCloud.hpp>
+#include <helpers/commonHelpers.hpp>
+#include <helpers/mathHelpers.hpp>
+
+class PointCloudTest : public ::testing::TestWithParam<int>
+{
+protected:
+	std::vector<rgl_field_t> allNotDummyFields = {XYZ_F32,     IS_HIT_I32,  RAY_IDX_U32,  ENTITY_ID_I32,  INTENSITY_F32,
+	                                              RING_ID_U16, AZIMUTH_F32, DISTANCE_F32, RETURN_TYPE_U8, TIME_STAMP_F64};
+
+	std::vector<rgl_field_t> fieldsWithPaddings = {PADDING_32,   XYZ_F32,        PADDING_16,     IS_HIT_I32,  PADDING_8,
```

Comment:
Well, padding is supposed to create padding in binary output when using FormatPointsNode, so probably the tests are verifying if padding behavior is correct.

The more concerning thing here is violating `Single Source of Truth` by defining `allNotDummyFields`, delusionally hoping that it will be updated when new fields are added. Not gonna happen. I'd move it to `RGLFields.hpp.` @nebraszka

Reply:
On point, moving to RGLFields.hpp may be.